compiler/testData/codegen/box/reflection/callBy/valueClasses/nonNullObject/defaultArguments/constructorWithInlineClassParameters.kt

```diff
@@ -0,0 +1,124 @@
+// TARGET_BACKEND: JVM
+// WITH_REFLECT
+
+import kotlin.test.assertEquals
+
+inline class A(val x: String)
```

Comment:
Please, use `@JvmInline value class` syntax for inline classes.

Reply:
I have fixed it.
ba8ba1d